doc/usecases/elektra_web/UC_finding_keys.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: Finding keys
+
+## Summary
+
+Title: Finding keys
+Scope: Configuration
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User searches for a key by filtering the key database.
+
+## Scenarios
+
+Precondition: View configuration of an instance
+Main success scenario: User searches for a key by entering information (its path
+  or data) in a field. The whole key database is filtered according to the
+  contents of the search field.
```

Comment:
No description what happens if (multiple) key(s) is/are found. Will it/they be shown in the tree?